diff --git a/Externals/curl/lib/vtls/mbedtls.c b/Externals/curl/lib/vtls/mbedtls.c
index ef0b9492a..832811e66 100644
--- a/Externals/curl/lib/vtls/mbedtls.c
+++ b/Externals/curl/lib/vtls/mbedtls.c
@@ -31,7 +31,7 @@
 
 #ifdef USE_MBEDTLS
 
-#include <mbedtls/net.h>
+#include <mbedtls/net_socket.h>
 #include <mbedtls/ssl.h>
 #include <mbedtls/certs.h>
 #include <mbedtls/x509.h>
diff --git a/Externals/mbedtls/library/error.c b/Externals/mbedtls/library/error.c
index a1cf83aed..d0401900d 100644
--- a/Externals/mbedtls/library/error.c
+++ b/Externals/mbedtls/library/error.c
@@ -101,7 +101,7 @@
 #endif
 
 #if defined(MBEDTLS_NET_C)
-#include "mbedtls/net.h"
+#include "mbedtls/net_socket.h"
 #endif
 
 #if defined(MBEDTLS_OID_C)
diff --git a/Externals/mbedtls/library/net.c b/Externals/mbedtls/library/net.c
index b5d06888b..1ce9f1b68 100644
--- a/Externals/mbedtls/library/net.c
+++ b/Externals/mbedtls/library/net.c
@@ -27,7 +27,7 @@
 
 #if defined(MBEDTLS_NET_C)
 
-#include "mbedtls/net.h"
+#include "mbedtls/net_socket.h"
 
 #include <string.h>
 
